src/sidebar/components/MenuKeyboardNavigation.js

```diff
@@ -110,18 +109,3 @@ export default function MenuKeyboardNavigation({
     </div>
   );
 }
-
-MenuKeyboardNavigation.propTypes = {
-  className: propTypes.string,
-
-  // Callback when the menu is closed via keyboard input
-  closeMenu: propTypes.func,
-
-  // When  true`, sets focus on the first item in the list
-  // which has a role="menuitem" attribute.
-  visible: propTypes.bool,
-
-  // Array of nodes which may contain <MenuItems> or any nodes
```

Comment:
The comment for `children` does not match that in `MenuKeyboardNavigationProps`, Generally speaking, I actually don't know our source of truth in these cases, but I'm included to just go with what we have for the jsdocs.

Reply:
The JSDoc is the only source of truth going forwards :)